.devcontainer/docker-compose-local.yaml

```diff
@@ -81,6 +81,9 @@ services:
       - 80:80
     environment:
       - DEPLOYMENT_TYPE=local
+      # The local file needs to be available through a mount,
+      # if persistence is needed
+      - SQLALCHEMY_DATABASE_URL=sqlite:///local.db
```

Comment:
Can't we have postgres vars removed & postgres removed from the docker-compose? If you connect to an external postgres changing the SQLALCHEMY_DATABSE_URL it would still work, right?

Reply:
IIRC there are dependencies to the postgres variables somewhere inside. I can make a quick test and try to find out where they are used. They were used only to create the db url, but I found errors when I removed them. I'll give it a check.